USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -287,7 +287,13 @@ public void createBookHeaders(string bookname)
 
             bookNameCount++;
         }
-
+        public void clearDocumentElements()
```

Comment:
I don't think this is necessary if you make the change to the tests that I recommended